app/templates/core/create-account-modal/confirmation-view.jade

```diff
@@ -0,0 +1,35 @@
+.modal-body
+  .modal-body-content
+    
+    h4.m-y-1(data-i18n="signup.account_created")
+    
+    .text-center.m-y-1
+      if view.signupState.get('path') === 'student'
+        p(data-i18n="signup.confirm_student_blurb")
+        
+      else
+        p(data-i18n="signup.confirm_individual_blurb")
+      
+    .signup-info-box-wrapper.m-y-3
+      .text-danger(data-i18n="signup.write_this_down")
+      .signup-info-box.text-center
+        if me.get('name')
+          h4
+            strong
+              span(data-i18n="general.username")
+              | : #{me.get('name')}
+        if me.get('email')
+          h5
+            strong
```

Comment:
Oh yeah, I think you did bring it up before, but old habits die hard! I switched it.